### PR TITLE
component selector colors

### PR DESCRIPTION
**Pull request by nelsonpecora**
[trello](https://trello.com/c/lgkRAUbR/28-page-layout-designation-on-selector)

screen shot 2016-08-25 at 12 40 54 pm

### CODE REVIEW

File: styleguide/component-select.scss [hunk 1]
```diff
@@ -101,49 +92,28 @@ $layout-parent-bg-color: $purple-25;
   justify-content: flex-start;
 }
 
+.selected-info {
+  border-left-width: 10px;
+  border-left-style: solid;
+  border-left-color: $layout-color;
```
Comment (davidandrewfrey) [inline styleguide/component-select.scss]: Minor stuff:
– alphabetical order
– could just set with `border-left` shorthand as well

Comment (nelsonpecora) [inline styleguide/component-select.scss]: we're explicitly overriding the color later, but good catch about the alpha

File: styleguide/component-select.scss [hunk 2]
```diff
@@ -101,49 +92,29 @@ $layout-parent-bg-color: $purple-25;
   justify-content: flex-start;
 }
 
+.selected-info {
+  // we're explicitly overriding the color below
```
Comment (davidandrewfrey) [inline styleguide/component-select.scss]: Yeah! It'll still do that with the shorthand. Just gave it a whirl to confirm:
http://codepen.io/davidandrewfrey/pen/wWLxNq

**Comment by davidandrewfrey** [comment]
⭐ ⭐ ⭐ ⭐ ⭐ 
Thank you for making the updates

**Pull request merged**